src/main/java/com/aetherteam/aether_genesis/blockTileEntity/HolyFurnaceBlockEntity.java

```diff
@@ -0,0 +1,24 @@
+package com.aetherteam.aether_genesis.blockTileEntity;
+
+import com.aetherteam.aether.blockentity.AbstractAetherFurnaceBlockEntity;
+import net.minecraft.core.BlockPos;
+import net.minecraft.network.chat.Component;
+import net.minecraft.world.entity.player.Inventory;
+import net.minecraft.world.inventory.AbstractContainerMenu;
+import net.minecraft.world.inventory.FurnaceMenu;
+import net.minecraft.world.item.crafting.RecipeType;
+import net.minecraft.world.level.block.state.BlockState;
+
+public class HolyFurnaceBlockEntity extends AbstractAetherFurnaceBlockEntity {
+    public HolyFurnaceBlockEntity(BlockPos pPos, BlockState pBlockState) {
+        super(GenesisBlockEntityTypes.HOLYSTONE_FURNACE.get(), pPos, pBlockState, RecipeType.SMELTING);
+    }
+
+    protected Component getDefaultName() {
+        return Component.translatable("container.furnace");
+    }
+
+    protected AbstractContainerMenu createMenu(int pId, Inventory pPlayer) {
+        return new FurnaceMenu(pId, pPlayer, this, this.dataAccess);
```

Comment:
The Holystone Furnace should use a custom menu with the blue frame to replace the yellow flame in the GUI.